curvefs/src/tools/curvefs_tool.h

```diff
@@ -196,32 +207,61 @@ class CurvefsToolRpc : public CurvefsTool {
      * as long as one succeeds, it returns true and ends sending
      */
     virtual bool SendRequestToServices() {
-        uint32_t failHostNumner = 0;
+        uint32_t failHostNumber = 0;
+        bool ret = false;
         for (const std::string& host : hostsAddr_) {
             SetController();
-            if (channel_->Init(host.c_str(), nullptr) != 0) {
-                std::cerr << "fail init channel to host: " << host << std::endl;
+            std::shared_ptr<brpc::ChannelOptions> channelOpt = nullptr;
```

Comment:
No need to allocate channel options on heap, a stack variable is enough.

Reply:
> No need to allocate channel options on heap, a stack variable is enough.

fix